src/webapi/node.rs

```diff
@@ -341,6 +341,18 @@ pub trait INode: IEventTarget {
             @{self.as_ref()}.normalize();
         }
     }
+
+    /// Element.getAttribute() returns the value of a specified attribute on the element.
+    /// If the given attribute does not exist, the value returned will either be
+    /// null or "" (the empty string);
+    ///
+    /// [(Javascript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute)
+    // https://dom.spec.whatwg.org/#ref-for-dom-element-getattribute
+    fn get_attribute( &self, name: &str ) -> Option< String > {
+        js!(
+            return @{self.as_ref()}.getAttribute( @{name} );
+        ).try_into().unwrap()
+    }
```

Comment:
We already have this method in `IElement::get_attribute`, and AFAIK the `get_attribute` is not part of the `Node` interface so this should be removed.

Any reason why did you add it here?

Reply:
Sorry my mistake. I skipped the help message and finally get it work now. So yes this shoud be removed